export_options.json

```diff
@@ -0,0 +1,16 @@
+[
```

Comment:
This file is used by svgexport, you can look at that package for how these different parameters are processed.